src/net/relay.rs

```diff
@@ -932,35 +937,44 @@ impl Relayer {
             // process blocks we downloaded
             let new_dled_blocks =
                 Relayer::preprocess_downloaded_blocks(&sort_ic, network_result, chainstate);
-            for new_dled_block in new_dled_blocks.into_iter() {
-                debug!("Received downloaded block for {}", &new_dled_block);
-                new_blocks.insert(new_dled_block);
+            for (new_dled_block_ch, block_data) in new_dled_blocks.into_iter() {
+                debug!(
+                    "Received downloaded block for {}/{}",
+                    &new_dled_block_ch,
+                    &block_data.block_hash()
+                );
```

Comment:
```suggestion
                debug!(
                    "Received downloaded block",
                    "consensus_hash" => %new_dled_block_ch,
                    "anchor_block_hash" => %block_data.block_hash(),
                );
```

Same suggestion for the other debug lines in the function too.

Reply:
Can I keep both?  I have scripts that expect `${consensus_hash}/${block_hash}`, since it's a pattern we use pervasively in the code.

Reply:
Yeah, that works.